src/constants.js

```diff
@@ -17,14 +17,19 @@ export function generatePrefixes (options) {
     waze: 'waze://',
     yandex: 'yandexnavi://',
     moovit: 'moovit://',
-    'yandex-maps': 'yandexmaps://maps.yandex.ru/'
-  }
+    'yandex-maps': 'yandexmaps://maps.yandex.ru/',
+  };
 }
 
-export function prefixForGoogleMaps (alwaysIncludeGoogle) {
-  return isIOS && !alwaysIncludeGoogle
+export function prefixForGoogleMaps(alwaysIncludeGoogle) {
+  switch (isIOS) {
+    case true : return isIOS && !alwaysIncludeGoogle
+    ? 'comgooglemaps://'
+    : 'https://www.google.com/maps';
+    case false :  return isIOS && !alwaysIncludeGoogle
     ? 'comgooglemaps://'
-    : 'https://maps.google.com/'
+    : 'https://maps.google.com/'; 
+  }
```

Comment:
I feel like this might be a slightly cleaner way:

```
if (isIOS && !alwaysIncludeGoogle) {
  return 'comgooglemaps://'
}
  
return isIOS
  ? 'https://www.google.com/maps'
  : 'https://maps.google.com/'
```

Reply:
Yes, that'd be much cleaner, didn't think of such an approach...